src/Compiler/Microsoft.CodeAnalysis.Razor/src/ComponentTagHelperDescriptorProvider.cs

```diff
@@ -512,7 +503,7 @@ private IEnumerable<(IPropertySymbol property, PropertyKind kind)> GetProperties
                     kind = PropertyKind.Ignored;
                 }
 
-                if (!property.GetAttributes().Any(a => SymbolEqualityComparer.Default.Equals(a.AttributeClass, symbols.ParameterAttribute)))
+                if (!property.GetAttributes().Any(a => a.AttributeClass.HasFullName(ComponentsApi.ParameterAttribute.MetadataName)))
```

Comment:
This doesn't seem generally correct (though it's an existing problem) as it doesn't match attributes derived from `ParameterAttribute`. That's probably ok, just pointing it out.